src/plugins/topbar/topbar.jsx

```diff
@@ -36,28 +72,52 @@ export default class Topbar extends React.Component {
     let inputStyle = {}
     if(isFailed) inputStyle.color = "red"
     if(isLoading) inputStyle.color = "#aaa"
+
+    const { urls } = getConfigs()
+    let control = []
+    let formOnSubmit = null
+
+    if(urls) {
+      let rows = []
+      urls.forEach((link, i) => {
+        rows.push(<option key={i} value={link.url} selected={i===this.state.selectedIndex}>{link.name}</option>)
```

Comment:
This causes a React warning, please fix: 

> Warning: Use the `defaultValue` or `value` props on <select> instead of setting `selected` on <option>.